src/app/add-new-product/add-new-product.component.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<body>
```

Comment:
**suggestion:** you should not have <html>, <body>, <!Doctype html>, <header> tags anywhere inside your components, this tags are core parts and already exist inside the `index.html` file (where your app-root component will eventually be built)